packages/story/src/stories/toggle/toggle.stories.tsx

```diff
@@ -2,6 +2,9 @@ import type { Meta, StoryObj } from "@storybook/react";
 import Component from "../../lib/Notion";
 import json from "./toggle.json";
 
+import { Toggle } from "../../../../core/src/lib";
+import type { ToggleArgs } from "../../../../core/src/lib";
```

Comment:
p1: don't import with relative path

it is mono repo

see: workspace property on root package.json

Reply:
I have resolved this issue.